gcs/src/helpers/mavlinkConstants.js

```diff
@@ -88,3 +89,205 @@ export const GPS_FIX_TYPES = [
+    frametypename: 'H',
+  },
+}
+
+export const FRAME_TYPE_MAP_OCTA_QUAD = {
+  0: {
+    motorOrder: [1, 6, 4, 7, 3, 8, 2, 5],
+    direction: ['CCW', 'CW', 'CW', 'CCW', 'CCW', 'CW', 'CW', 'CCW'],
+    frametypename: 'Plus',
+  },
+  1: {
+    motorOrder: [1, 6, 4, 7, 3, 8, 2, 5],
+    direction: ['CCW', 'CW', 'CW', 'CCW', 'CCW', 'CW', 'CW', 'CCW'],
+    frametypename: 'X',
+  },
+  2: {
+    motorOrder: [1, 6, 4, 7, 3, 8, 2, 5],
+    direction: ['CCW', 'CW', 'CW', 'CCW', 'CCW', 'CW', 'CW', 'CCW'],
+    frametypename: 'V',
+  },
+  3: {
+    motorOrder: [1, 6, 4, 7, 3, 8, 2, 5],
+    direction: ['CCW', 'CW', 'CW', 'CCW', 'CCW', 'CW', 'CW', 'CCW'],
+    frametypename: 'H',
+  },
+}
+
+export const FRAME_TYPE_MAP_Y6 = {
+  0: {
+    motorOrder: [5, 1, 6, 4, 2, 3],
+    direction: ['CW', 'CCW', 'CCW', 'CW', 'CW', 'CCW'],
+    frametypename: 'X',
+  },
+  10: {
+    motorOrder: [1, 2, 3, 4, 5, 6],
+    direction: ['CW', 'CCW', 'CW', 'CCW', 'CW', 'CCW'],
+    frametypename: 'Y6B',
+  },
+  11: {
+    motorOrder: [2, 5, 1, 4, 3, 6],
+    direction: ['CCW', 'CW', 'CCW', 'CW', 'CCW', 'CW'],
+    frametypename: 'Y6F',
+  },
+}
+
+export const FRAME_TYPE_MAP_DODECA_HEXA_COPTER = {
+  0: {
+    motorOrder: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+    direction: [
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+    ],
+    frametypename: 'Plus',
+  },
+  1: {
+    motorOrder: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+    direction: [
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+      'CCW',
+      'CW',
+      'CW',
+      'CCW',
+    ],
+    frametypename: 'X',
+  },
+}
+
+export const FRAME_CLASS_MAP = {
+  0: { name: 'Undefined', frametype: false, numberOfMotors: 4 },
+  1: { name: 'Quad', frametype: FRAME_TYPE_MAP_QUAD, numberOfMotors: 4 },
+  2: { name: 'Hexa', frametype: FRAME_TYPE_MAP_HEX, numberOfMotors: 6 },
+  3: { name: 'Octa', frametype: FRAME_TYPE_MAP_OCTA, numberOfMotors: 8 },
+  4: {
+    name: 'OctaQuad',
+    frametype: FRAME_TYPE_MAP_OCTA_QUAD,
+    numberOfMotors: 8,
+  },
+  5: { name: 'Y6', frametype: FRAME_TYPE_MAP_Y6, numberOfMotors: 6 },
+  6: { name: 'Heli', frametype: false, numberOfMotors: 1 },
+  7: { name: 'Tri', frametype: false, numberOfMotors: 3 },
+  8: { name: 'Single Copter', frametype: false, numberOfMotors: 1 },
+  9: { name: 'CoaxCopter', frametype: false, numberOfMotors: 2 },
+  10: { name: 'BiCopter', frametype: false, numberOfMotors: 2 },
+  11: { name: 'Heli_Dual', frametype: false, numberOfMotors: 2 },
+  12: {
+    name: 'DodecaHexa',
+    frametype: FRAME_TYPE_MAP_DODECA_HEXA_COPTER,
+    numberOfMotors: 12,
+  },
+  13: { name: 'HeliQuad', frametype: false, numberOfMotors: 4 },
+  14: { name: 'Deca', frametype: false, numberOfMotors: 10 },
+}
+
+export const LETTERS = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
```

Comment:
If the frame types have frames with 12 motors, then shouldn't the `LETTERS` array include 12 letters? Also can you rename this to something more clear, e.g. `MOTOR_LETTER_LABELS`?